Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM spotify/techdocs
+
+RUN apk --no-cache add bash nodejs npm
```

Comment:
Are node & npm not already on the image? I would have thought they would need to be for the cli to actually work.

Reply:
Seems node is there, but npm isn't! I tried both and neither